### ISSUE TITLE
Restarting level does not update personal best

### ISSUE THREAD

**Issue by vasama** [vittorioromeo/SSVOpenHexagon#371]

**Comment by vittorioromeo** [comment]
Are you saying that after getting a personal best, pressing restart does not save it?

**Comment by vasama** [comment]
Yes. See this video: https://www.youtube.com/watch?v=ZHAPLXNqJII
The personal best is initially 0. I play until ~3.5s then restart, and exit after ~0.8s. Only the second result is saved.

**Comment by vittorioromeo** [comment]
> Yes. See this video: https://www.youtube.com/watch?v=ZHAPLXNqJII The personal best is initially 0. I play until ~3.5s then restart, and exit after ~0.8s. Only the second result is saved.

Ah, but you did not die. Personal best is saved only upon death. Maybe I should change that, but no one ever complained about it before. Why would you restart if you have not died?

**Comment by vasama** [comment]
My restart reflex is too fast. I sometimes do it just before dying. 😛

**Comment by vittorioromeo** [comment]
I see, I will look into performing the personal best checks during a restart as well.
